source/dotnet/Library/AdaptiveCards/Rendering/AdaptiveHostConfig.cs

```diff
@@ -148,7 +148,6 @@ public static AdaptiveHostConfig FromJson(string json)
             }
             catch (JsonException ex)
             {
-                Debugger.Break();
```

Comment:
Ha! nice catch!

Reply:
found while debugging :D

---
In reply to: [278344312](https://github.com/Microsoft/AdaptiveCards/pull/2560#discussion_r278344312) [](ancestors = 278344312)